zerver/models.py

```diff
@@ -63,6 +69,7 @@ def flush_per_request_caches():
 @cache_with_key(lambda *args: display_recipient_cache_key(args[0]),
                 timeout=3600*24*7)
 def get_display_recipient_remote_cache(recipient_id, recipient_type, recipient_type_id):
+    # type: (int, int, int) -> List[Dict[str, Any]]
```

Comment:
The return type should be something like `Union[str, List[Dict[str, Any]]]` because [Line 80](https://github.com/zulip/zulip/pull/759/commits/d1b460f1f71cb5c4edca04f4b97a01d38444c0c0#diff-c29a639b9c074638e5b08d59f0be3997R80) returns a string.